### ISSUE TITLE
Column name parsing fail for multiple-dot names

### ISSUE THREAD

**Issue by hombit** [lincc-frameworks/nested-pandas#144]
**Bug report**

I have a column name "R.A." and `lsdb.from_dataframe()` fails

```
File /ocean/projects/phy210048p/malanche/lsdb-tests/cenv/lib/python3.11/site-packages/nested_pandas/nestedframe/core.py:58, in NestedFrame._is_known_hierarchical_column(self, colname)
     56 """Determine whether a string is a known hierarchical column name"""
     57 if "." in colname:
---> 58     left, right = colname.split(".")
     59     if left in self.nested_columns:
     60         return right in self.all_columns[left]

ValueError: too many values to unpack (expected 2)
```

We probably should count dots or unpack it safer, e.g. with `left, right, *others = colname.split(".")`

**Before submitting**
Please check the following:

- [x] I have described the situation in which the bug arose, including what code was executed, information about my environment, and any applicable data others will need to reproduce the problem.
- [x] I have included available evidence of the unexpected behavior (including error messages, screenshots, and/or plots) as well as a descriprion of what I expected instead.
- [x] If I have a solution in mind, I have provided an explanation and/or pseudocode and/or task list.

**Comment by dougbrn** [comment]
issue pertains to query wrapping (adding for search discoverability)

**Comment by gitosaurus** [comment]
Note that this particular bug appears to have been fixed by @dougbrn in commit 1cde0c10f8985ad6b6da468edeba5b3263f89a18.  #177 changed the method, and didn't regress.
